### ISSUE TITLE
other data sources

### ISSUE THREAD

**Issue by sckott** [ropenscilabs/laselva#7]
* germany
* france
* italy

non-national programs:

* japan
* australia

**Comment by kunstler** [comment]
- Australia http://onlinelibrary.wiley.com/doi/10.1890/14-0458R.1/abstract

- Japan http://db.cger.nies.go.jp/JaLTER/ER_DataPapers/archives/2011/ERDP-2011-01/metadata

- Germany I need to check the url with a German colleague in the coming weeks. It seems that there are two different url one for the data between 1987 (only former west Germany) and 2002 (here Https://gdi.ti.bund.de/geonetwork/srv/eng/metadata.show?uuid=adbde42d-0a65-40f3-8c36-b6d5b3359d62
 or Https://gdi.ti.bund.de/wo/bwi/bwi2009_download.php) and one for the data of 2002 and 2012 at https://bwi.info/Download/de/ (It seems that the data are only available in ms access2003 not sure what is the best way to read that in R across different OS. I have no code for that) 

- Spain IFN3 http://www.mapama.gob.es/en/biodiversidad/servicios/banco-datos-naturaleza/informacion-disponible/ifn3_bbdd_descargas.htm.aspx

- Spain IFN2 http://www.mapama.gob.es/en/biodiversidad/servicios/banco-datos-naturaleza/informacion-disponible/ifn2_descargas.aspx
I have some code but not from already processed data

- Italy https://www.inventarioforestale.org/en/node/13 need logging

- France http://inventaire-forestier.ign.fr/spip/spip.php?article532 need logging 
(I have a R code formatting the downloaded data but this need to be turn in a proper function https://github.com/kunstler/FrenchNFI)

I will try to work on that in the coming weeks and

**Comment by kunstler** [comment]
It is probably worth to check recent paper analyses large collection of forest inventory data

Liang, J. et al. 2016. Positive biodiversity-productivity relationship predominant in global forests. - Science 354: aaf8957-aaf8957.

Stephenson, N. L. et al. 2014. Rate of tree carbon accumulation increases continuously with tree size. - Nature 507: 90–93.

And this paper may provide additional data
http://onlinelibrary.wiley.com/doi/10.1111/jvs.12419/abstract

**Comment by kunstler** [comment]
There is also all the CTFS forestGEO plots
some of them are open (but the website seems down)

http://www.ctfs.si.edu/plots/

**Comment by kunstler** [comment]
For tropical forest this network is probably also important https://www.forestplots.net/en
but not public...

**Comment by sckott** [comment]
thanks for all this @kunstler ! will look into the sources you don't mention having code for already

**Comment by kunstler** [comment]
There are several smaller data set in Ecology data papers (apparently ESA is moving the data to Figshare ...)

- one from my lab: http://onlinelibrary.wiley.com/doi/10.1002/ecy.1759/full
- India http://www.esapubs.org/archive/ecol/E092/115/default.htm
- Australia (mentioned above) http://onlinelibrary.wiley.com/doi/10.1890/14-0458R.1/abstract

**Comment by kunstler** [comment]
I'm not sure this belong to the same package, but a recent data paper provide harmonized data from all EU forest inventory (see http://www.nature.com/articles/sdata2016123 , data on figshare https://figshare.com/collections/A_high-resolution_pan-European_tree_occurrence_dataset/3288407). This not 'dynamic' data with re-measurement of tree diameter, only species presence absence in plot (in two size classes), but the coverage is quite large. 

![image](https://cloud.githubusercontent.com/assets/4505188/23711487/59ce535e-0420-11e7-9ad6-3ea05f1c4ced.png)

This probably a data base that can complement 'dynamic' NFI data.

**Comment by sckott** [comment]
@kunstler very nice - seems like we can utilize that since it's on figshare

**Comment by sckott** [comment]
@kunstler any progress on this:

> France http://inventaire-forestier.ign.fr/spip/spip.php?article532 need logging
(I have a R code formatting the downloaded data but this need to be turn in a proper function https://github.com/kunstler/FrenchNFI)

**Comment by sckott** [comment]
@kunstler What's the diff. between IFN2 and IFN3 data for Spain?

**Comment by kunstler** [comment]
IFN2 and IFN3 are different census, so we we need both (IFN4 is starting to be available for some provinces)

**Comment by kunstler** [comment]
No... Sorry to be slow with that. I will try to have it done before the end of the year

**Comment by sckott** [comment]
@kunstler okay, thanks for the updates

**Comment by sckott** [comment]
notes form https://onlinelibrary.wiley.com/doi/full/10.1111/jvs.12419

- 🛑 splot  - locked down
- 🛑 European Vegetation Archive - locked down
- 🛑 Vegbank - some data is open, but no way that I can see that it can be used programmatically - no direct links to datasets, they hide link generation behind javascript - locked down enough to be inaccessible  
- GIVD seems to just be a listing of datasets
- New Zealand mentioned and their forest inventory. Looked - most promising was https://nvs.landcareresearch.co.nz/Data/Search - there seems to be some open data there, but not sure which plots are part of the official inventory

**Comment by kunstler** [comment]
Hi Scott, I hope everything is going well for you. I just wanted to let you know that in a recently published paper (arising from the sDIV sAPROPOS working group you attended) we published with Sophia and other co-authors some forest inventory data for Sweden and Finland in this dryad repo (with blurred geographic coordinates) https://datadryad.org/stash/dataset/doi:10.5061/dryad.wm37pvmkw . This might be interesting to add to the package?

**Comment by sckott** [comment]
Thanks @kunstler for letting me know!  Doing well as can be living in the u.s. Hope you are doing well.

Definitely will add

**Comment by SDupire** [comment]
Hi, as discussed with Georges, here is the link to the Chile NFI : https://ifn.infor.cl/

They make a difference between native forest (bosque nativo) and plantations. 

I can give a hand for the understanding if needed as everything is in Spanish

**Comment by pughtam** [comment]
Hi,
Netherlands NFI is also open access, although I don't know if online to download.
Finland also has some data online now: NFI12: https://opendata.luke.fi/dataset/national-forest-inventory-nfi-vmi12
(for NFI11: https://opendata.luke.fi/dataset/national-forest-inventory-11-field-plot-data)
Susanne Suvanto has made a translation from Finnish.
Cheers,
Tom

**Comment by pughtam** [comment]
Seems that those links no longer work. These ones instead:
NFI12: https://etsin.fairdata.fi/dataset/58751432-d60f-40dd-a68f-2fa0708d47d2
NFI11: https://etsin.fairdata.fi/dataset/bcc94a6d-ab7d-4959-aded-ebf026aa2a53

**Comment by kunstler** [comment]
The NFI from the Netherlands seems to be available here https://www.probos.nl/publicaties/overige/1094-mfv-2006-nbi-2012
